python/cudf/cudf/core/dataframe.py

```diff
@@ -3161,24 +3161,29 @@ def from_pandas(cls, dataframe, nan_as_null=True):
 
         df = cls()
         # Set columns
-        for i, colk in enumerate(dataframe.columns):
+        for colk in dataframe.columns:
             vals = dataframe[colk].values
```

Comment:
As of Pandas 0.24 this is the recommended path and is guaranteed to be zero copy:
```suggestion
            vals = dataframe[colk].array
```